temporal-sdk/src/main/java/io/temporal/worker/WorkerFactory.java

```diff
@@ -216,7 +216,9 @@ public synchronized void start() {
 
     // Workers check and require that Temporal Server is available during start to fail-fast in case
     // of configuration issues.
-    workflowClient.getWorkflowServiceStubs().connect(null);
```

Comment:
`connect` was already calling `getServerCapabilities` but then made a gRPC health check call after which is unnecessary and failed with API keys since it didn't have a namespace